tests/basic/customer_issues.go

```diff
@@ -0,0 +1,352 @@
+			for _, eachVol := range vols {
+				volumesPresent = append(volumesPresent, eachVol)
+			}
+		}
+
+		// Check if the cluster consists of 10 nodes
+		log.InfoD("Get all nodes present in the cluster")
+
+		allNodes := []node.Node{}
+		// create an array with storage and storage less nodes added
+		for _, each := range node.GetStorageNodes() {
+			allNodes = append(allNodes, each)
+		}
+
+		for _, each := range node.GetStorageLessNodes() {
+			allNodes = append(allNodes, each)
+		}
+
+		// Verify total nodes available is 10
+		dash.VerifyFatal(len(allNodes) == 10, true, "required minimum of 10 nodes for the test to run")
+
+		var nodesSplit1 = []node.Node{}
+		var nodesSplit2 = []node.Node{}
+		if len(node.GetStorageLessNodes()) != 4 && len(node.GetStorageNodes()) > 6 {
+			nodesSplit1 = allNodes[0:4]
+			nodesSplit2 = allNodes[5:10]
+		} else {
+			nodesSplit1 = node.GetStorageNodes()
+			nodesSplit2 = node.GetStorageLessNodes()
+		}
+
+		var getKvdbLeaderNode node.Node
+		allkvdbNodes, err := GetAllKvdbNodes()
+		log.FailOnError(err, "Failed to get list of KVDB nodes from the cluster")
+
+		for _, each := range allkvdbNodes {
+			if each.Leader {
+				getKvdbLeaderNode, err = node.GetNodeDetailsByNodeID(each.ID)
+				log.FailOnError(err, "Unable to get the node details from NodeID [%v]", each.ID)
+				break
+			}
+		}
+
+		log.InfoD("KVDB Leader node is [%v]", getKvdbLeaderNode.Addresses)
+		var allStorageExceptKVDB []node.Node
+		for _, each := range nodesSplit1 {
+			if each.Id != getKvdbLeaderNode.Id {
+				allStorageExceptKVDB = append(allStorageExceptKVDB, each)
+			}
+		}
+
+		// Create 2 zones from the Storage and storageless Node
+		zone1 := []node.Node{}
+		zone2 := []node.Node{}
+
+		zone1StorageEle, zone2StorageEle := getRandomNumbersFromArrLength(len(allStorageExceptKVDB), len(allStorageExceptKVDB)/2)
+		for _, each := range zone1StorageEle {
+			zone1 = append(zone1, allStorageExceptKVDB[each])
+		}
+		for _, each := range zone2StorageEle {
+			zone2 = append(zone2, allStorageExceptKVDB[each])
+		}
+
+		zone1StorageLessEle, zone2StorageLessEle := getRandomNumbersFromArrLength(len(nodesSplit2), len(nodesSplit2)/2)
+		for _, each := range zone1StorageLessEle {
+			zone1 = append(zone1, nodesSplit2[each])
+		}
+		for _, each := range zone2StorageLessEle {
+			zone2 = append(zone2, nodesSplit2[each])
+		}
+
+		flushiptables := func() {
+			if !iptablesflushed {
+				// Flush all iptables from all the nodes available forcefully
+				for _, eachNode := range allNodes {
+					log.InfoD("Flushing iptables rules on node [%v]", eachNode.Name)
+					log.FailOnError(flushIptableRules(eachNode), "Iptables flush all failed on node [%v]", eachNode.Name)
+				}
+			}
+		}
+
+		// force flush iptables on all the nodes at the end
+		defer flushiptables()
+
+		volumeState, err := GetVolumesInDegradedState(contexts)
+		if err != nil {
+			fmt.Println("Error not Nil")
```

Comment:
remove these print statements

Reply:
removed the complete section , thanks for pointing out , this was added as a debug statement